tensorflow/stream_executor/BUILD

```diff
@@ -707,6 +715,17 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "redzone_allocator",
```

Comment:
Desirably I want the `redzone_allocator` target to incorporate both `redzone_allocator_cuda` and `redzone_allocator_rocm`, but both implementation files are in child folders. I'm not sure if bazel allows that?

Besides, `if_cuda_is_configured` and `if_rocm_is_configured` is not yet available in this file. Will have to pull that utility in from `@local_config_rocm//rocm:build_defs.bzl` if we decided to go down this path.

Reply:
Thanks, will do.